src/components/pages/IDPManagement/IDPList.tsx

```diff
@@ -126,6 +133,17 @@ export const IDPList = () => {
     }
   }
 
+  const renderStatusColor = (status: string) => {
+    if (status === IdpAccountStatus.ACTIVE) return 'confirmed'
+    else if (status === IdpAccountStatus.DISABLED) return 'declined'
+    else if (
+      status === IdpAccountStatus.OPEN ||
+      status === IdpAccountStatus.IDP_CREATED
+    )
+      return 'pending'
+    else return 'label'
```

Comment:
Check my comments from other PR: https://github.com/eclipse-tractusx/portal-frontend/pull/967/files#r1709533449
use the types from the beginning where we receive the data - in the apiSlice and then create a map instead of a function.

This code has been written before this change but let's take the time to improve it.

Reply:
Changes Done